spec/fixtures/site_settings/locale_override.yml

```diff
@@ -0,0 +1,14 @@
+basic:
+  default_locale:
+    default: 'en'
+    enum: 'LocaleSiteSetting'
+
+search:
+  min_search_term_length:
+    client: true
+    default:
+      development: 1
```

Comment:
Here it is: https://meta.discourse.org/t/sitesetting-file-format-update-for-locale-override-and-hidden-params-based-on-rails-env/66254